fuzzing/utils/fuzz_method_utils.go

```diff
@@ -32,3 +35,24 @@ func IsPropertyTest(method abi.Method, prefixes []string) bool {
 	}
 	return false
 }
+
+// IsAssertionTest checks whether the method is configured by the attached fuzzer to be a target of assertion testing.
+// Returns true if this target should be tested, false otherwise.
+func isAssertionTest(method abi.Method, includeViewMethods bool) bool {
+	// Only test constant methods (pure/view) if we are configured to.
+	return !method.IsConstant() || includeViewMethods
+}
+
+// BinTestByType sorts a contract's methods by whether they are assertion, property, or optimization tests.
+func BinTestByType(contract *compilationTypes.CompiledContract, testCfg config.TestingConfig) (assertionTests, propertyTests, optimizationTests []abi.Method) {
```

Comment:
for utility functions like these, I try to minimize the amount of package dependencies. so i'm going to remove the dependency on the `config` package by explicitly passing in the prefixes, etc.